notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -43,33 +43,90 @@
     "## Setup"
```

Comment:
<p>I guess this is remaining here unintentionally.</p><p>(<code>seaborn</code> is included in the requirements.txt)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Nope! Good catch! Removing it.